Workaround to lag issue TCLProject#14

DrParadox7 opened the pull request:
Allow skipping the lag heavy part of `onEntityLiving()` if the spells reliant on it are disabled.

Related to #14.

DrParadox7 committed:
Workaround to lag issue #14

Allow skipping the lag heavy part of onEntityLiving() if the spells reliant on it are disabled.

TCLProject commented:
This workaround is smart and would work, but vortex logic is also handled in the same block. I will be making a new dev branch shortly, it has changes that should drastically reduce the lag from this function. If need be, you can copypaste some things from there

DrParadox7 committed:
OnEntityLiving() lag reduction

Disable tasks in `onEntityLiving()` should the features relying on it are disabled to reduce lag.
Added Spatial Vortex config.

DrParadox7 commented:
Made new changes with some copypasta from your dev branch.
